python/flask/correctness/flask-route-decorator.py

```diff
@@ -0,0 +1,50 @@
+from flask import Blueprint
+
+blueprint = Blueprint()
+
+
+# ok: Only 1 decorator
```

Comment:
Semgrep reports this test syntax as a valid rule id. The `ok:` test syntax expects to be followed by a ruleid. If you want to indicate why this is ok, you can do that on a separate line

```suggestion
# Only 1 decorator
# ok: flask-route-decorator
```